### ISSUE TITLE
Left-over concepts

### ISSUE THREAD

**Issue by AnnikaTjuka** [concepticon/concepticon-data#877]
We need a new check on left-over concepts that we could map if we have more lists to express them. There are some nice terms we MAY find in other lists, but feel that they may be too much to be added now. So the idea would be: for 2.3 or a later version, we could make a concepticon sprint, where we hunt down unlinked concepts during one week of labor.

> first mentioned by @LinguList in PR [#699](https://github.com/concepticon/concepticon-data/pull/699#pullrequestreview-335186056)

**Comment by CarolinHu** [comment]
Suggested by @MacyL and @LinguList on PR #882:

- [ ] BOULDER (i.e. huge pieces of rock) 
- [ ] COME (COMMAND) (although it would have to be decided how to deal with 3555 COME HERE)

The SIL/ Swadesh-Rowe lists also often include:
- [ ] MIRITI PALM

**Comment by CarolinHu** [comment]
I opened a PR that adds ASTRINGENT, BROAD BEAN, and HANDSPAN. The concepts I proposed in #791 can be left aside for now as I couldn't find any more instances for these. Some "problem cases" remain in this issue:

- RIVER DEER - I'm not sure what the conclusion is after your explanations, @MacyL, and how the description for RIVER DEER should look like/how encompassing it needs to be. Should this be discussed in a separate issue?

- COME (COMMAND) - this was proposed here https://github.com/concepticon/concepticon-data/pull/882#issuecomment-659327365
However, we also have the concept COME HERE. Should they be collapsed, or should a relation be introduced such as COME (COMMAND) narrower COME HERE?

- ALSO - if this is to be added, should it be a broader concept that generally expresses "in addition" and include some instances of "too" as well? (The "toos" would have to be checked one by one I think, to make sure it is really the "as well" meaning, not the "too" as in "too narrow")
